packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -46,6 +43,20 @@ if ([production, development].indexOf(mode) < 0) {
     throw new Error(`Invalid mode "${mode}"`)
 }
 
+if (pkg?.ccExtensibility?.extendable && pkg?.ccExtensibility?.extends) {
+    const extendsAsArr = Array.isArray(pkg?.ccExtensibility?.extends)
+        ? pkg?.ccExtensibility?.extends
+        : [pkg?.ccExtensibility?.extends]
+    const conflicts = extendsAsArr.filter((x) => pkg?.ccExtensibility?.extendable?.includes(x))
+    if (conflicts?.length) {
+        throw new Error(
+            `Dependencies in 'extendable' and 'extends' cannot overlap, fix these: ${conflicts.join(
+                ', '
+            )}"`
+        )
+    }
+}
+
```

Comment:
check to make sure the user doesn't put `retail-react-app` in both `extendable` and also `extends` as the two conflict

Reply:
I can't seem to find any uses of this "extendable" configuration property in any of the projects. Do we need it? Or is this future thinking work?

Reply:
"extendable" makes self-referential paths like `retail-react-app/*` work from within `retail-react-app` when developing